idunn/api/places_list.py

```diff
@@ -4,28 +4,44 @@
 
 from idunn.utils.settings import Settings
 from idunn.utils.index_names import IndexNames
+from idunn.utils.categories import Categories
 from idunn.places import POI
 from idunn.api.utils import fetch_bbox_places, LONG, SHORT, DEFAULT_VERBOSITY_LIST
 
 from apistar import http
 
 from pydantic import BaseModel, ValidationError, validator
-from typing import List, Optional
+from pydantic.error_wrappers import ErrorWrapper
+from typing import List, Optional, Any
 
 logger = logging.getLogger(__name__)
 
 VERBOSITY_LEVELS = [LONG, SHORT]
 
-MAX_WIDTH = 1.0 # max bbox longitude in degrees
+MAX_WIDTH = 1.0 # max bbox longitude in degrees
 MAX_HEIGHT = 1.0  # max bbox latitude in degrees
 
+ALL_CATEGORIES = None
```

Comment:
The purpose of this global variable is to access it into the validators easily

Reply:
instead of using a component, it would be more simple to read the categories from file here